From d3460e971c48e4d368e4cedfbb7dc25224b43c18 Mon Sep 17 00:00:00 2001
From: "Lu, Shenghua" <shenghua.lu@intel.com>
Date: Wed, 5 Dec 2012 17:35:29 +0800
Subject: [PATCH 29/38] [PORT FROM MAIN] [UX] Add some variables in psb_drm.h
 for cursor A control

BZ:52060

Add some variables for cursor A appear/disappear, position,
configuration setting.

Change-Id: I8fb157d0fb5cf44706c251d09df8c37b90187ed8
Signed-off-by: Shenghua Lu <shenghua.lu@intel.com>
Reviewed-on: http://android.intel.com:8080/81963
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 libc/kernel/common/linux/psb_drm.h |   10 ++++++++++
 1 files changed, 10 insertions(+), 0 deletions(-)

diff --git a/libc/kernel/common/linux/psb_drm.h b/libc/kernel/common/linux/psb_drm.h
index 853658d..095c1fc 100644
--- a/libc/kernel/common/linux/psb_drm.h
+++ b/libc/kernel/common/linux/psb_drm.h
@@ -694,6 +694,16 @@ struct drm_psb_register_rw_arg {
  } sprite;
  uint32_t subpicture_enable_mask;
  uint32_t subpicture_disable_mask;
+ struct {
+/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
+ uint32_t CursorADDR;
+ uint32_t xPos;
+ uint32_t yPos;
+ uint32_t CursorSize;
+/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
+ } cursor;
+ uint32_t cursor_enable_mask;
+ uint32_t cursor_disable_mask;
 };
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
 enum {
-- 
1.7.9

